client/src/modules/Course/Components/ReviewModal.tsx

```diff
@@ -79,25 +77,8 @@ const ReviewModal = ({
 
   useEffect(() => {
     setAllowSubmit(valid.professor && valid.major && valid.grade && valid.text);
-    if (isLoggedIn) {
-      getNoReviews();
-    }
   }, [valid]);
 
-  /**
-   * Determines if the current user has no reviews, so they should receive
-   * the anonymous modal
-   */
-  async function getNoReviews() {
-    const response = await axios.post('/api/profiles/count-reviews', {
-      netId
-    });
-    const res = response.data;
-    if (response.status === 200) {
-      setNoReviews(res.result === 0);
-    }
-  }
-
   function onProfessorChange(newSelectedProfessors: string[]) {
```

Comment:
This can be part of a codebase cleanup PR, but it'd be nicer if these functions were all arrow functions (i.e. `const onProfessorChange = (newSelectedProfessors: string[]) => {...}`).

Reply:
I only converted the four onXChange methods -- should I convert the other functions as well?

Reply:
Yeah, that would be great!